cluster/charts/crossplane/templates/rbac-manager-clusterrole.yaml

```diff
@@ -89,4 +89,22 @@ rules:
   - patch
   - watch
   - delete
+- apiGroups:
+  - ""
+  resources:
+  - namespaces/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - pkg.crossplane.io
+  resources:
+  - providerrevisions/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions/finalizers
```

Comment:
Are you sure you mean `customresourcedefinitions/finalizers` instead of 
`compositeresourcedefinitions/finalizers` ? If so, could you explain why rbac-manager needs this?